source/common/stats/symbol_table_impl.cc

```diff
@@ -177,7 +174,8 @@ std::string SymbolTableImpl::decodeSymbolVec(const SymbolVec& symbols) const {
 
 void SymbolTableImpl::incRefCount(const StatName& stat_name) {
   // Before taking the lock, decode the array of symbols from the SymbolTable::Storage.
-  SymbolVec symbols = SymbolEncoding::decodeSymbols(stat_name.data(), stat_name.dataSize());
+  SymbolVec symbols =
+      SymbolTableImpl::Encoding::decodeSymbols(stat_name.data(), stat_name.dataSize());
```

Comment:
I think this and other callsites are overqualified, i.e. could just be `Encoding::decodeSymbols(...)`. This is a very small nit but I'm curious what's good practice?